site/content/docs/single-sign-on/_index.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
@ianaya89 do you know why this page is not showing on the sidebar under "Single sign-on", in the docs? Played around with it but couldn't figure it out yet.